src/main/java/dev/lambdaurora/lambdabettergrass/metadata/LBGLayerType.java

```diff
@@ -87,7 +87,7 @@ public static void forEach(Consumer<LBGLayerType> consumer) {
 	public static void load(Identifier resourceId, ResourceManager resourceManager) {
 		var id = new Identifier(resourceId.getNamespace(), resourceId.getPath().replace(".json", ""));
 		try {
-			var stream = resourceManager.method_14486(resourceId).getInputStream();
+			var stream = resourceManager.open(resourceId);
```

Comment:
This probably should become `try (var stream = resourceManager.open(resourceId)) {`